pkg/lib/web/embedded_resource.go

```diff
@@ -101,12 +101,15 @@ func (m *GlobalEmbeddedResourceManager) watch() {
 				break
 			}
 
-			if event.Op&fsnotify.Write != fsnotify.Write && event.Op&fsnotify.Create != fsnotify.Create {
-				break
+			switch event.Op {
```

Comment:
Refactor so that `watcher.Add` and `watcher.Remove` only appear in one method only. The initial setup code and the `watch()` function should use this method.